requirements.txt

```diff
@@ -13,3 +13,4 @@ pandas
 plotly
 largestinteriorrectangle
 pydicom
+petsc4py
```

Comment:
Based on the github actions, github does not manage to install DarSIA including petsc4py. Can you have a look at the installation report? Anything needed in addition?